docs/source/user/interface.rst

```diff
@@ -161,6 +161,20 @@ Toggle Simple Interface mode using the View menu:
 When you leave Simple Interface mode, the original layout of the main
 area is restored.
 
+Searching
+---------
+
+JupyterLab has an advanced built-in search plugin for finding text within a
+notebook or other document, which uses the `Ctrl-F` shortcut by default.
```

Comment:
In RST file, you need to use double tick to get the text highlighted in mono space and in other places the `+` seems to be used rather than `-` to point to combine keys shortcut:

```suggestion
notebook or other document, which uses the ``Ctrl+F`` (``Cmd+F`` for macOS) shortcut by default.
```

The shortcut is actually Cmd-F for Apple.